com.zoho.Notebook.yaml

```diff
@@ -0,0 +1,42 @@
+app-id: com.zoho.Notebook
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+command: notebook
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=session-bus
```

Comment:
this is not allowed. You need to determine which dbus name the app needs and add them individually.

Reply:
Alright!